interfaces/devdoc/threadpool_requirements.md

```diff
@@ -60,30 +60,32 @@ MOCKABLE_FUNCTION(, THREADPOOL_HANDLE, threadpool_create, EXECUTION_ENGINE_HANDL
 
 **NON_THREADPOOL_01_002: [** If `execution_engine` is `NULL`, `threadpool_create` shall fail and return `NULL`. **]**
 
+**N_O_N_THREADPOOL_01_011: [** `threadpool_create` shall perform any actions needed to open the `threadpool` object. **]**
+
```

Comment:
I don't think we need this as there is no "open" per se any longer - we are deprecating it.

Reply:
Consider removing please.

Reply:
Done